docs/posts/unified-stream-batch/index.qmd

```diff
@@ -0,0 +1,419 @@
+---
+title: "Stream-batch unification through Ibis lens"
+author: "Chloe"
```

Comment:
can you add your last name? despite my previous shenanigans we should be consistent